build/houdini-scenarios.yml

```diff
@@ -0,0 +1,37 @@
+# Houdini scenarios
+
+parameters:
+- name: profile
+  type: string
+  default: ''
+- name: warmup
+  type: number
+  default: 5
+- name: duration
+  type: number
+  default: 15
+- name: connection
+  type: string
+  default: ''
+
+
+# Scenarios
+- name: scenarios 
+  type: object
+  default: 
+  - displayName: Houdini EchoTodo
+    arguments: --scenario echotodo $(houdiniJobs) --application.channel edge --application.framework net6.0
```

Comment:
I don't want it in a separate table, too much work. Can you put it with the other MVC benchmarks instead? they are also using bombardier so it will be easier also for the charts. The other plaintext/json ones should be kept where you put them.